lib/DDG/Spice/Justdeleteme.pm

```diff
@@ -0,0 +1,30 @@
+package DDG::Spice::Justdeleteme;
```

Comment:
Your filename/package name should mimic the ID. We typically use CamelCase for packages.

If you id is `just_delete_me` it's expected that the package name is `DDG::Spice::JustDeleteMe` -- I prefer this package name for legibility. Can you change the package name and `id` throughout the code to match the IA page?

I would suggest you copy these files somewhere and remove them from the repo. Then run `duckpan new JustDeletMe` and duckpan will create the correct files for you. You can then paste in the content